text/0000-named-tracers.md

```diff
@@ -0,0 +1,69 @@
+# Named Tracers
+
+**Status:** `proposed`
+
+_Creating tracers using a factory mechanism and naming those tracers in accordance with the library that provides the instrumentation for a traced component._
+
+## Suggested reading
+
+* [Proposal: Tracer Components](https://github.com/open-telemetry/opentelemetry-specification/issues/10)
+* [Global Instance discussions](https://github.com/open-telemetry/opentelemetry-specification/labels/global%20instance)
+
+## Motivation
+
+Instead of having a global tracer singleton, every instrumentation library needs to create its own tracer, which also has a proper name that identifies the library / logical component that uses this tracer. If there are two different instrumentation libraries for the same technology (e.g. MongoDb), these instrumentation libraries should have distinct names.
+
+This proposal originates from an `opentelemetry-specification` proposal on [components](https://github.com/open-telemetry/opentelemetry-specification/issues/10), but the term "components" was quickly found to being too generic and possibly misleading, since it is already used frequently in differing contexts and connotations.
+
+The purpose of _Named Tracers_ is the ability to allow for a more fine-grained control of the amount of tracing data being generated by a library or application. Associating a name (e.g. _"io.opentelemetry.contrib.mongodb"_) with a tracer allows for a logical coupling of this tracer and a certain technology or library (like _MongoDB_ in this example).
```

Comment:
In a default implementation (SDK), should the name of the tracer be incorporated in the Span name or should be a property on the Span? Same question for Metrics as well. I think incorporating this in the name of the metrics makes a lot of sense because then the metric name must be unique inside the component.

Reply:
The tracer name _could_ be a property on the Span, which would be a good way to e.g. address the "tracer component issue" (https://github.com/open-telemetry/opentelemetry-specification/issues/10).
However, this is deliberately left out of this RFC's scope, since it  is neither a requirement nor a consequence of it.